app/poros/map.rb

```diff
@@ -0,0 +1,13 @@
+class Map
+  attr_reader :distance, :travel_time, :direction_instructions
+
+  def initialize(map_data)
+    @distance = map_data[:distance].round(1)
+    if map_data[:realTime] <= 60
+      @travel_time = '1 min'
+    else
+      @travel_time = "#{map_data[:realTime]/60} min #{map_data[:realTime]%60} sec"
+    end
+    @direction_instructions = map_data[:legs][0][:maneuvers].map { |direction| direction[:narrative] }.join(" ")
+  end
+end
```

Comment:
I like the conditions you have in here. This is a log of logic that's in your initialize method, which we want to avoid. We could do somethign like this instead: 

```ruby
class Map
  attr_reader :distance, :direction_instructions

  def initialize(map_data)
    @distance = map_data[:distance].round(1)
    @realtime = map_data[:realTime]
    @direction_instructions = map_data[:legs][0][:maneuvers].map { |direction| direction[:narrative] }.join(" ")
  end

   def travel_time 
         if real_time<= 60
            return   '1 min'
          else
            return "#{map_data[:realTime]/60} min #{map_data[:realTime]%60} sec"
    end
   end 
end
```